### PR TITLE
'Use Wallet Balance' remains to display wallet amount of DAI when switch to 'Position' option

### LINKED ISSUE #654: 'Use Wallet Balance' remains to display wallet amount of DAI when switch to 'Position' option
Related to #583, #513

**Steps:** 
Connect to a wallet
open Split position page
select the 'Position' option

**AR:** 'Use Wallet Balance' remains to display wallet amount of DAI 

**ER:** 0.00 should be displayed in the  'Use Wallet Balance' button (and it should be disabled) until select a position to split

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #654

### CODE REVIEW

File: src/pages/SplitPosition/Form.test.tsx [hunk 1]
```diff
@@ -65,35 +65,6 @@ test('show unlock button with zero allowance', async () => {
   expect(unlockBtn).toBeInTheDocument()
 })
 
-test('toggle unlock button visiblity according to allowance and amount', async () => {
```
Comment (mariano-aguero) [inline src/pages/SplitPosition/Form.test.tsx:68]: This test, was not update since a long time, and throw an error, check for some stuff that doesn't exist

Comment (mariano-aguero) [reply src/pages/SplitPosition/Form.test.tsx:68]: In the same field, we can see some lines commented , so ...

File: src/pages/SplitPosition/Form.test.tsx [hunk 2]
```diff
@@ -65,35 +65,6 @@ test('show unlock button with zero allowance', async () => {
...
-        allowanceMethods={allowanceMethods}
-        collateral={defaultToken}
-        onCollateralChange={onCollateralChange}
-        splitPosition={splitPosition}
-        tokens={tokens}
-      />
-    )
-  )
-
-  const unlockBefore = findByTestId('unlockButton')
```
Comment (mariano-aguero) [inline src/pages/SplitPosition/Form.test.tsx:86]: This doesn't exist

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr700--conditionaltokensexplorer.review.gnosisdev.com

**Review by elena-zh — APPROVED** [review]
LGTM!
Thanks!

**Pull request merged**